### PR TITLE
Django migration 3.1 to 4.1

### PR DESCRIPTION
**Pull request by khanchi97**

### CODE REVIEW

File: jet/dashboard/dashboard.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from django.template.loader import render_to_string
 from jet.dashboard import modules
 from jet.dashboard.models import UserDashboardModule
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```
Comment (khanchi97) [inline jet/dashboard/dashboard.py:10]: `ugettext(), ugettext_lazy(), ugettext_noop(), ungettext(), and ungettext_lazy()` are deprecated in favor of the functions that they’re aliases for: `gettext(), gettext_lazy(), gettext_noop(), ngettext(), and ngettext_lazy()`.

File: jet/dashboard/dashboard_modules/google_analytics.py [hunk 2]
```diff
@@ -16,14 +16,14 @@
 import httplib2
 from jet.dashboard.modules import DashboardModule
 from oauth2client.client import flow_from_clientsecrets, OAuth2Credentials, AccessTokenRefreshError, Storage
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
 from django.conf import settings
-from django.utils.encoding import force_text
+from django.utils.encoding import force_str
```
Comment (khanchi97) [inline jet/dashboard/dashboard_modules/google_analytics.py:21]: `smart_text() and force_text()` are deprecated in favor of `smart_str() and force_str()`.

File: docs/dashboard_custom_module.rst [hunk 3]
```diff
@@ -140,7 +140,7 @@ in ``dashboard_modules_views.py`` file inside your application:
 
     .. code-block:: python
 
-        from django.conf.urls import url
+        from django.urls import re_path as url
```
Comment (khanchi97) [inline docs/dashboard_custom_module.rst:143]: `django.conf.urls.url` is deprecated in favor of `django.urls.re_path`

File: jet/ordered_set.py [hunk 4]
```diff
@@ -1,11 +1,11 @@
 import collections
 
 
-class OrderedSet(collections.MutableSet):
+class OrderedSet(collections.abc.MutableSet):
```
Comment (khanchi97) [inline jet/ordered_set.py:4]: `MutableSet` exists under `collections.abc` now.

**Review by laveesingh — APPROVED** [review]

**Pull request merged**